Prototyper/Classes/PrototyperController.swift

```diff
@@ -144,17 +147,19 @@ open class PrototyperController: NSObject {
     }
     
     // MARK: Share App
-    
     private static func shareApp() {
-        guard let instantiatedViewController = UIStoryboard(name: "Share",
-                                                            bundle: Bundle(for: ShareViewController.self)).instantiateInitialViewController() else {
-            return
-        }
+        let instantiatedViewController = UIHostingController(rootView: ShareView().environmentObject(Model()))
```

Comment:
Using a shared Model as an `environmentObject` is an good idea, I would suggest to use something like the `PrototypeSettings` and the `APIHandler` to check if a user is logged in and so on.